src/frontend/components/users.js

```diff
@@ -0,0 +1,47 @@
+import anitha from '../image/anitha.jpg'
+import kamila from '../image/kamila.jpg'
+import kainy from '../image/kainy.png'
+import florin from '../image/florin.jpg'
+import selchuck from '../image/selchuck.jpg'
+
+const database = [
+  
+        {
+            id: '1',
+            name: 'Anitha Dharneedharan ',
+            Language: 'JavaScript',
+            Location: 'Sutton',
```

Comment:
Can we change the locations of the `users` in order to have a more dynamic view of the results?